### ISSUE TITLE
Renewing. distribution certificate

### ISSUE THREAD

**Issue by Popolechien** [kiwix/kiwix-apple#1011]
I got a notification from Apple that our distribution certificate would expire in about a month, but when I go to the Certificates, Identifiers & Profiles page it seems we've maxed out our certification already, with no option for renewal / extension

![Image](https://github.com/user-attachments/assets/c0f72f0f-432d-47ef-b646-dc1ca1a2d17d)

Thoughts / suggestions?

**Comment by BPerlakiH** [comment]
It seems to me, that we have reached the limit of max 3 distribution certificates.
@rgaudin can you have a look please which one is used on CI/CD, I don't have access to the secrets.

I think we can remove some of those certificates, that are not really in use (I just don't want to accidentally delete the wrong one).

**Comment by rgaudin** [comment]
![Image](https://github.com/user-attachments/assets/9b482474-cdd3-432a-8ccc-4b3e5f259fc4)

@BPerlakiH, the situation is unclear because:

- App Store Auth Key and Development certificate dates back from that time when I created the Distribution certificate
- There are a bunch of org-level secrets created in January (by you).

I believe those are your attempts at using the signing repo.

Given we don't explicitly include a certificate in CI to sign for Distribution but use the App Store Key instead, I'm not sure which is actually used when signing.

My suggestion would be to remove both those you created and see if the workflow fails. If it does, we'll remove the one I created and create a new one. If it works, then we'll renew mine. WDYT?
